MatchmakeSample/MatchmakeSample/Program.cs

```diff
@@ -104,8 +104,28 @@ private static async Task CreateMatchmakeTicket(Player player, string mmQueueNam
             {
                 QueueName = mmQueueName,
                 GiveUpAfterSeconds = 15,
-                Creator = new MatchmakingPlayer { Entity = new PlayFab.MultiplayerModels.EntityKey { Id = player.context.EntityId, Type = player.context.EntityType } } // Why isn't this just the caller?
+                Creator = new MatchmakingPlayer
+                {
+                    Entity = new PlayFab.MultiplayerModels.EntityKey { Id = player.context.EntityId, Type = player.context.EntityType }
+                }
             };
+
+            // uncomment and edit the following lines if you want to use Multiplayer Servers allocation
```

Comment:
This comment makes it seem like the below should be commented by default?

Reply:
ouch, thx